Examples: CMake Error

Compiling without having CUDA present and only with Alpaka sometimes leads to the CMake error in the `example/`:
```
CMake Error at isaac-1.3.1/lib/ISAACConfig.cmake:154 (list):
list sub-command REMOVE_DUPLICATES requires list to be present.
```

Maybe one needs a check if CUDA or ALPAKA+CUDA is used or simply if the list exists.

--- a/lib/ISAACConfig.cmake
+++ b/lib/ISAACConfig.cmake
@@ -151,7 +151,9 @@ if (ISAAC_ALPAKA)
     endif()
 endif()
 
-list(REMOVE_DUPLICATES CUDA_NVCC_FLAGS)
+if (CUDA_NVCC_FLAGS)
+    list(REMOVE_DUPLICATES CUDA_NVCC_FLAGS)
+endif()
 
 
 ################################################################################

